contracts/api3-server-v1/proxies/CompositeApi3ReaderProxyV1.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+import "./interfaces/ICompositeApi3ReaderProxyV1.sol";
+import "../../interfaces/IApi3ReaderProxy.sol";
+
+import "hardhat/console.sol";
+
+/// @title An immutable proxy contract that is used to read a composite data feed
+/// by multiplying two IApi3ReaderProxy data feeds
+/// @dev This contract implements the AggregatorV2V3Interface to be compatible with
+/// Chainlink aggregators. This allows the contract to be used as a drop-in
+/// replacement for Chainlink aggregators in existing dApps.
+/// Refer to https://github.com/api3dao/migrate-from-chainlink-to-api3 for more
+/// information about the Chainlink interface implementation.
+contract CompositeApi3ReaderProxyV1 is ICompositeApi3ReaderProxyV1 {
```

Comment:
What about the other use cases? My thinking is to have a complete proxy composition library. Even if there is no use case now, I want to avoid having another iteration like this when the use case happens.

Reply:
> My thinking is to have a complete proxy composition library

FWIW I regretted every time I broke a contract down into multiple contracts

Reply:
I would guess that depends on the purpose of the breaking down. I think here it's justified.